src/pm/hydra/tools/bootstrap/external/slurm_query_node_list.c

```diff
@@ -35,12 +289,12 @@ static HYD_status group_to_nodes(char *str)
     *tmp = 0;   /* remove the closing ']' */
 
     /* Find the number of sets */
-    tnodes = HYDU_strdup(nodes);
+    tnodes = MPL_strdup(nodes);
     tmp = strtok(tnodes, ",");
     for (i = 1; tmp; i++)
         tmp = strtok(NULL, ",");
 
-    HYDU_MALLOC(set, char **, i * sizeof(char *), status);
+    HYDU_MALLOC_OR_JUMP(set, char **, i * sizeof(char *), status);
```

Comment:
@raffenet should I also fix this?

Reply:
Yeah, it doesn't really belong in this series. Good catch.